src/Documentation/RuleSetDocumentationGenerator.php

```diff
@@ -44,7 +44,7 @@ public function generateRuleSetsDocumentation(RuleSetDescriptionInterface $defin
 
         $title = "Rule set ``{$definition->getName()}``";
         $titleLine = str_repeat('=', \strlen($title));
-        $doc = "{$titleLine}\n{$title}\n{$titleLine}\n\n".$definition->getDescription();
+        $doc = "{$titleLine}\n{$title}\n{$titleLine}\n\n".RstUtils::ensureProperInlineCode($definition->getDescription());
```

Comment:
I believe this change causes wrong formatting of URLs in rule sets (see PR's diff).